src/examples/echoAsync/echo.go

```diff
@@ -0,0 +1,63 @@
+// This is a echo console running on the device UART.
+// Connect using default baudrate for this hardware, 8-N-1 with your terminal program.
+package main
+
+import (
+	"machine"
+	"time"
+)
+
+// change these to test a different UART or pins if available
+var (
+	uart = machine.UART0
+	tx   = machine.UART_TX_PIN
+	rx   = machine.UART_RX_PIN
+)
+
+func keepAlive() {
+	for {
+		time.Sleep(10 * time.Millisecond)
+	}
+}
+
+func main() {
+	machine.LED.Configure(machine.PinConfig{Mode: machine.PinOutput})
+	time.Sleep(time.Second)
+	time.Sleep(time.Second)
+	time.Sleep(time.Second)
+	time.Sleep(time.Second)
+	time.Sleep(time.Second)
+	time.Sleep(time.Second)
+	time.Sleep(time.Second)
+	time.Sleep(time.Second)
+
+	//uart.Configure(machine.UARTConfig{TX: tx, RX: rx})
```

Comment:
I do not quite understand why this prevented it from working. Can someone explain what is going on?

Reply:
Probably because `UART0` is already configured as part of the runtime setup on most platforms. Otherwise we cannot do any `println()` statements on panic etc.